### PR TITLE
vim-patch:75ab590f8504

### PR DESCRIPTION
**Pull request by clason**
Update runtime files
https://github.com/vim/vim/commit/75ab590f8504a8912ca0b8c58f6b897bb7a34f07

### CODE REVIEW

**Review by zeertzjq — COMMENTED** [review]
- `ExitPre` should be moved above `FileAppendCmd`
- Missing several `exe 'sign unplace '` -> `call sign_unplace()` changes

**Comment by zeertzjq** [comment]
The commit message should probably mention that `expand()` change depends on patch 8.2.4726.

**Comment by clason** [comment]
> The commit message should probably mention that `expand()` change depends on patch 8.2.4726.

Which change is that?

**Comment by seandewar** [comment]
The skipped change for `expand('<script>')`

**Review by seandewar — COMMENTED** [review]
Termdebug doc changes should be added to `nvim_terminal_emulator.txt`.

Also, I think the order of our events is better than Vim's (maybe we could move `DiffUpdated` after `DiffChangedPre` to keep it sorted, though?)

**Comment by zeertzjq** [comment]
What is `DiffChangedPre`?

**Comment by clason** [comment]
> Termdebug doc changes should be added to nvim_terminal_emulator.txt.

Done.

> Also, I think the order of our events is better than Vim's (maybe we could move DiffUpdated after DiffChangedPre to keep it sorted, though?)

It's `**Dir**ChangedPre`, though ;)

**Comment by seandewar** [comment]
Derp :laughing:

File: runtime/pack/dist/opt/termdebug/plugin/termdebug.vim [hunk 1]
```diff
@@ -1378,16 +1391,17 @@ func s:HandleCursor(msg)
       endif
       exe lnum
       normal! zv
-      exe 'sign unplace ' . s:pc_id
-      exe 'sign place ' . s:pc_id . ' line=' . lnum . ' name=debugPC file=' . fname
```
Comment (seandewar) [inline runtime/pack/dist/opt/termdebug/plugin/termdebug.vim:1382]: prio was added in 8.2.1556 which isnt ported, but should be fine

**Review by seandewar — APPROVED** [review]
LGTM I think

File: runtime/pack/dist/opt/termdebug/plugin/termdebug.vim (hunk 1 above)
Comment (clason) [reply runtime/pack/dist/opt/termdebug/plugin/termdebug.vim:1382]: Since it was already used in the old line 1485 below, I would think so, too ;)

**Review by zeertzjq — APPROVED** [review]

**Pull request merged**